src/main/java/ui/components/PanelMenuBar.java

```diff
@@ -85,6 +86,7 @@ private HBox createNameArea() {
                 }
             }
         });
+        Tooltip.install(nameArea, new Tooltip("Rename"));
```

Comment:
Tool tips can be slightly more detailed. Sometimes just a one word may not be enough.